features/step_definitions/registry.rb

```diff
@@ -385,8 +386,16 @@
 end
 
 Given /^default image registry route is stored in the#{OPT_SYM} clipboard$/ do |cb_name| 
+  ensure_admin_tagged
+  ensure_destructive_tagged
```

Comment:
I don't think this operation is destructive or can interfere with other operations. Can we remove the `ensure_destructive` call?

Reply:
Yeah, you are correct. Tag is removed.